microservice/index.js

```diff
@@ -72,19 +73,32 @@ app.post('/tree', async (req, res) => {
       && req.body.uuid !== ""){
       duplicate  = await data.checkForExistingTree(req.body.uuid);
     }
+    console.log(`capture in field data schema for ${req.body.uuid} has duplicates ${duplicate === null}`);
     if(duplicate !== null){
       res.status(200).json({ duplicate });
     } else {
-      const tree = await data.createTree( user.id, req.body.device_identifier, req.body);
-      console.log("created tree " + tree.uuid);
-      res.status(201).json({ tree });
+       // translate to field-data capture api
```

Comment:
I really wanted to move this api invocation to the cron, but that might miss the user and duplicate lookups done here in `index.js`. Longer term, cron should use field-data service directly. For now, this works.